src/azure-cli/azure/cli/command_modules/acr/_params.py

```diff
@@ -324,6 +330,18 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
         c.argument('password1', options_list=['--password1'], help='Flag indicating if first password should be deleted', action='store_true', required=False)
         c.argument('password2', options_list=['--password2'], help='Flag indicating if second password should be deleted.', action='store_true', required=False)
 
+    with self.argument_context('acr agentpool') as c:
+        c.argument('registry_name', options_list=['--registry', '-r'])
+        c.argument('agent_pool_name', options_list=['--name', '-n'], help='The name of the agentpool.')
+        c.argument('tag', help='The tag of the agentpool.')
+        c.argument('count', type=int, help='The count of the agentpool.')
+        c.argument('tier', help='The tier of the agentpool.')
```

Comment:
Should we add more description about the tier here e.g S1 maps to D2_V3 (2CPU, 4GB Memory)

Reply:
Or we need a option_list here too.